seapath_ros_driver/include/seapath_socket.hpp

```diff
@@ -8,17 +8,18 @@
 #include <cstring>
 #include <unistd.h>
 #include <vector>
+#include <sys/time.h>
 
-
-class SeaPathSocket {
+class seapath_socket {
```

Comment:
This is a class, thus should be in PascalCase (SeapathSocket), same with the constructor.